global.json

```diff
@@ -139,7 +139,7 @@
       }]
     }, {
       "id": "pushstarter_project",
-      "priority": 1,
+      "priority": 0.96,
```

Comment:
order bounces around with the hello world project, since they have the same priority. altered so it's no 2 on list.